wezterm-gui/src/termwindow/mouseevent.rs

```diff
@@ -839,14 +839,12 @@ impl super::TermWindow {
             MouseCursor::Hand
         } else if pane.is_mouse_grabbed() || outside_window {
```

Comment:
FWIW, I think you you will want to allow the shape to override in the grabbed-mouse case. That is the most likely situation where this escape sequence will be used.

Reply:
Ah right, I missed that in the initial recommendation, thank you for catching that!
Fixed in 11cf8b68ba43689f11dd27a26101b9bee552e609